protocol/x/subaccounts/keeper/transfer_test.go

```diff
@@ -673,27 +674,47 @@ func TestTransferInsuranceFundPayments(t *testing.T) {
 		expectedInsuranceFundBalance        int64
 	}{
 		"success - send to insurance fund module account": {
+			perpetual:                           constants.BtcUsd_SmallMarginRequirement,
 			insuranceFundBalance:                2500,
 			subaccountModuleAccBalance:          600,
 			quantums:                            big.NewInt(500),
 			expectedSubaccountsModuleAccBalance: 100,  // 600 - 500
 			expectedInsuranceFundBalance:        3000, // 2500 + 500
 		},
 		"success - send from insurance fund module account": {
+			perpetual:                           constants.BtcUsd_SmallMarginRequirement,
 			insuranceFundBalance:                2500,
 			subaccountModuleAccBalance:          600,
 			quantums:                            big.NewInt(-500),
 			expectedSubaccountsModuleAccBalance: 1100, // 600 + 500
 			expectedInsuranceFundBalance:        2000, // 2500 - 500
 		},
 		"success - can send zero payment": {
+			perpetual:                           constants.BtcUsd_SmallMarginRequirement,
 			insuranceFundBalance:                2500,
 			subaccountModuleAccBalance:          600,
 			quantums:                            big.NewInt(0),
 			expectedSubaccountsModuleAccBalance: 600,
 			expectedInsuranceFundBalance:        2500,
 		},
+		"success - send to isolated insurance fund account": {
```

Comment:
nit: Can you also add a test-case for failure to send from the insurance fund for an isolated market due to lacking funds?